packages/jellyfish-api-core/__tests__/category/wallet.test.ts

```diff
@@ -631,4 +631,21 @@ describe('masternode', () => {
       })
     })
   })
+
+  describe('dumpPrivKey', () => {
+    it('should reveal private key of given address', async () => {
+      await waitForExpect(async () => {
+        const address = await client.wallet.getNewAddress()
+        const privateKey = await client.wallet.dumpPrivKey(address)
+
+        expect(typeof privateKey).toStrictEqual('string')
```

Comment:
its recommended to expect a exact output
refer https://github.com/DeFiCh/jellyfish/blob/35a5f074c1d1789f40733360baedf5aee2aa7534/packages/testcontainers/src/testkeys.ts

Reply:
Those test keys are different from what i am getting. Is that normal?